types/textfit/index.d.ts

```diff
@@ -3,7 +3,10 @@
 // Definitions by: Muhun Kim <https://github.com/x86chi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function textFit(element: HTMLElement, options?: textFit.TextFitOption): void;
+declare function textFit(
+    els: HTMLElement | HTMLElement[] | NodeListOf<Element> | HTMLCollection | null,
```

Comment:
this should also accept jQuery object, should it be supported here?
`els` cannot be a null at any point in the current implementation, there is not nullable expression kind of check at any point:
https://github.com/STRML/textFit/blob/master/textFit.js#L61

Reply:
Thanks for review with suggestion! Your suggestion has been apply to new commit. Can you review again?